public/app/percona/backup/components/StorageLocations/RemoveStorageLocationModal/RemoveStorageLocationModal.test.tsx

```diff
@@ -5,7 +5,7 @@ import { WarningBlock } from '../../../../shared/components/Elements/WarningBloc
 import { RemoveStorageLocationModal } from './RemoveStorageLocationModal';
 import { CheckboxField } from '@percona/platform-core';
 
-xdescribe('RemoveStorageLocationModal', () => {
+describe('RemoveStorageLocationModal', () => {
```

Comment:
This tests will likely fail. You can try to find a solution if you haven't already or just ignore if cannot find one. (`useField` problem that we talked)

Reply:
Ok it passed just ignore me then.